dev_scripts_helpers/notebooks/test/test_extract_notebook_images.py

```diff
@@ -0,0 +1,52 @@
+import logging
```

Comment:
I wouldn't check in the pngs.

We can assume that if there is a file, the file is correct, since we assume that nbconvert is correct